cms/static/js/spec/video/transcripts/videolist_spec.js

```diff
@@ -1,13 +1,13 @@
 define(
     [
         'jquery', 'underscore',
+        'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers',
         'js/views/video/transcripts/utils',
         'js/views/video/transcripts/metadata_videolist', 'js/models/metadata',
         'js/views/abstract_editor',
-        'common/js/spec_helpers/ajax_helpers',
         'xmodule'
     ],
-function ($, _, Utils, VideoList, MetadataModel, AbstractEditor, AjaxHelpers) {
+function ($, _, AjaxHelpers, Utils, VideoList, MetadataModel, AbstractEditor) {
```

Comment:
Good point. I generally follow a variant of the Python rule about imports:

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#imports-order

@pdesjardins what do you think about putting such a rule into the JavaScript style guide too?